demos/serverless/src/handlers.js

```diff
@@ -203,6 +203,20 @@ async function ensureLogStream(cloudWatchClient, logStreamName) {
   return null;
 }
 
+exports.create_log_stream = async (event, context, callback) => {
+  const body = JSON.parse(event.body);
+  if (!body.meetingId || !body.attendeeId) {
+    return response(400, 'application/json', JSON.stringify({error: 'Need properties: meetingId, attendeeId'}));
+  }
+  const logStreamName = `ChimeSDKMeeting_${body.meetingId.toString()}_${body.attendeeId.toString()}`;
+  const cloudWatchClient = new AWS.CloudWatchLogs({ apiVersion: '2014-03-28' });
+  await cloudWatchClient.createLogStream({
+    logGroupName: logGroupName,
+    logStreamName: logStreamName,
+  }).promise();
+  return response(200, 'application/json', JSON.stringify({}));
```

Comment:
Correct me here if this is needed, will it be better to have some data in response, like if the stream successfully got created or for some reason errors out.

Reply:
We just use the response status in the demo app to verify successful creation of the stream name.
The handler has logic to create a stream if it does not exist once the app stats emitting logs to cloudwatch.